packages/app/src/components/Root/Root.tsx

```diff
@@ -75,11 +81,23 @@ const SidebarLogo: FC<{}> = () => {
   );
 };
 
+const handleSearch = (query: string): void => {
+  // XXX (@koroeskohr): for testing purposes
+  // eslint-disable-next-line no-console
+  console.log(query);
+};
+
 const Root: FC<{}> = ({ children }) => (
   <SidebarPage>
     <Sidebar>
       <SidebarLogo />
-      <SidebarSpacer />
+      <SidebarSearchField onSearch={handleSearch} />
+      <SidebarDivider />
+      {/* Global nav, not org-specific */}
+      <SidebarItem icon={HomeOutlinedIcon} to="/" text="Home" />
+      <SidebarItem icon={HomeOutlinedIcon} to="/login" text="Plugins" />
+      <SidebarItem icon={AddCircleOutlineIcon} to="/create" text="Create..." />
+      {/* End global nav */}
       <SidebarDivider />
       <SidebarItem icon={HomeIcon} to="/" text="Home" />
       <SidebarItem icon={ExploreIcon} to="/explore" text="Explore" />
```

Comment:
Explore should be part of the Global nav (and replace the current Plugins link).